src/features/PeopleList/styled.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+
+export const Content = styled.section`
+    max-width: 1368px;
```

Comment:
Consider that this does not account for padding in children. In your solution movie list without padding is not the same width as on Figma. This is not only different from UI design but also from our other pages. 

![figma_example](https://user-images.githubusercontent.com/98344543/176731894-d4c5b4d2-ee81-46e8-8b61-1a660dbf922e.JPG)
![react_example](https://user-images.githubusercontent.com/98344543/176731887-96e335cd-80f3-4afa-8cdf-e238b9a7ded6.JPG)

Reply:
removed the padding, now it's the same as in figma I think